broken CI:  codspeed-benchmarks

### Bug Description

ref: https://github.com/sparckles/Robyn/actions/runs/11304482293/job/31442927639

```
  Error: Error Only Ubuntu 20.04 and 22.04 are supported at the moment
  Error: Process completed with exit code 1.
```

### Steps to Reproduce

_No response_

### Your operating system

None

### Your Python version (`python --version`)

None

### Your Robyn version

None

### Additional Info

_No response_

--- a/.github/workflows/codspeed.yml
+++ b/.github/workflows/codspeed.yml
@@ -11,7 +11,7 @@ on:
 
 jobs:
   benchmarks:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5

